infrastructure/modules/armonik/ingress-cert.tf

```diff
@@ -131,16 +181,30 @@ resource "local_sensitive_file" "ingress_ca" {
   file_permission = "0600"
 }
 
+resource "local_sensitive_file" "ingress_client_ca" {
+  count           = length(tls_self_signed_cert.client_root_ingress)
+  content         = tls_self_signed_cert.client_root_ingress.0.cert_pem
+  filename        = "${path.root}/generated/certificates/ingress/client_ca.crt"
+  file_permission = "0600"
+}
+
 resource "local_sensitive_file" "ingress_client_crt" {
   count           = length(tls_locally_signed_cert.ingress_client_certificate)
```

Comment:
You should use a `for_each` with the `tls_locally_signed_cert.ingress_client_certificate` map